ppml/services/bigdl-kms/docker/Dockerfile

```diff
@@ -0,0 +1,88 @@
+ARG BIGDL_VERSION=2.1.0-SNAPSHOT
+ARG SPARK_VERSION=3.1.2
+
+# jdk & scala
+FROM ubuntu:20.04 as scala
+ARG SCALA_VERSION=2.12
+ARG SCALA_SUB_VERSION_NUM=10
+ARG JDK_VERSION=8u192
+ARG JDK_URL=your_jdk_url
+ENV SCALA_VERSION ${SCALA_VERSION}
+ENV SCALA_SUB_VERSION_NUM ${SCALA_SUB_VERSION_NUM}
```

Comment:
do we need scala?

Reply:
Yes, since bigdl kms frontend (a service listener) is implemented in scala.